wordpress-mobile/WordPress-Android
# Issue/13066 ppn move add category action

Part of #13066 

This PR removes the "Add New Category" link from the select categories bottom sheet and adds a "+" to the toolbar/title area. It also removes "bolding" from the Add Category button on the Add Category view.

Before|After|Before|After
---|---|---|---
<img width="480" src="https://user-images.githubusercontent.com/506707/96194480-73b7f700-0f18-11eb-9202-25a09d2ba0a2.png">|<img width="480" src="https://user-images.githubusercontent.com/506707/96469870-b40ac400-11fb-11eb-8ec6-a5c2b1feba00.png">|<img width="480" src="https://user-images.githubusercontent.com/506707/96469916-c1c04980-11fb-11eb-99ee-d2591b8c3b65.png">|<img width="480" src="https://user-images.githubusercontent.com/506707/96469969-cedd3880-11fb-11eb-9914-4b97685ea8df.png">


**Merge Instructions**
Note: This PR will be merged into a feature branch
Make sure #13167 has been merged (this PR may need a sync after 13167 has been merged - let me know)
Remove the Not Ready For Merge label
Merge as normal

**To test:**

- Launch the app
- Select Blog posts
- Select Edit on a post
- From the edit post view, tap the UPDATE action to launch the bottom sheet
- Tap the categories row to launch the Select categories view
- Notice the "Add new category" link is gone and there is a "+" in the toolbar area
-Tap on the "+" to ensure the Add Category view is launched
- Notice the "Add Category" button text is no longer bold

PR submission checklist:
